src/XCCDF_POLICY/xccdf_policy_remediate.c

```diff
@@ -24,7 +24,11 @@
 #include <string.h>
 #include <errno.h>
 #include <sys/stat.h>
+
+#if defined(unix) || defined(__unix__) || defined(__unix)
```

Comment:
you keep repeating this conditional. perhaps unify all the checks and define your own in compat.h

ifdef OSCAP_WIN, OSCAP_UNIX, etc...